nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/lambda/PutLambda.java

```diff
@@ -155,33 +150,38 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
-        final AWSLambdaClient client = getClient(context);
+        final LambdaClient client = getClient(context);
 
         try {
             final ByteArrayOutputStream baos = new ByteArrayOutputStream();
             session.exportTo(flowFile, baos);
```

Comment:
It looks like these lines can be removed since the `session.read()` method now populates the payload.
```suggestion
```

Reply:
Good catch -- thanks